docs/rst/user_manual/dynamic_data_types.rst

```diff
@@ -0,0 +1,112 @@
+.. include:: ../exports/alias.include
+
+.. _dynamic_data_types_sec:
+
+########################################
+Configure Dynamic Data Types in Fast DDS
+########################################
+
+|ddsvisualizer| uses
+`Dynamic Types <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dynamic_types/dynamic_types.html>`_ in order
+to be able to introspect the data type and read the internal data.
+If the application does not know the data type, it will not be able to connect to such topic.
+There are two ways of retrieving the data type information to the application:
+
+********************************
+Automatically discover Data Type
+********************************
+
+In order for |ddsvisualizer| to discover automatically the data type of a topic,
+the |fastdds| publisher that may interact with the application must be configured to send this information.
+There are two main configurations that has to be taken into account:
+
+
+Generate DataType
+=================
+
+In order for Fast DDS to be able to retrieve the data type description, it must have the TypeInformation.
+In case that the |fastdds| application is already using the *Dynamic Types* API, this TypeInformation will be
+already available for the :code:`DomainParticipant`.
+
+Otherwise, if the data type is generated via
+`Fast DDS Gen <https://fast-dds.docs.eprosima.com/en/latest/fastddsgen/introduction/introduction.html>`_,
+it is required to generate it with option :code:`-typeobject`.
+This generates two auxiliar cpp files that include the type information of the data type.
+If doing so, the data must be registered in the |fastdds| application before creating the publisher,
```

Comment:
**If doing so,** the data...  -> sounds strange, I'd use any of the alternatives:
* **Then,** the data...
* **After that,** the data...
* **When this is done,** the data...